src/css/_higlass.scss

```diff
@@ -2,7 +2,7 @@
 .vitessce-container .higlass-wrapper {
   // https://sass-lang.com/documentation/at-rules/import#nesting
   // https://sass-lang.com/documentation/at-rules/import#importing-css
-  @import "~higlass/dist/hglib";
+  @import "../../node_modules/higlass/dist/hglib";
```

Comment:
`esbuild-plugin-sass` couldn't seem to resolve the ~ alias. I changed it to a relative import and the webpack dev-server seemed to still work (it also made the esbuild plugin happy).